docs/src/complexity_measures.md

```diff
@@ -11,3 +11,17 @@ complexity_normalized
 ReverseDispersion
 distance_to_whitenoise
 ```
+
+## Approximate entropy
+
+```@docs
+ApproxEntropy
+```
+
+## Convenience functions
+
+We provide a few convenience functions for widely used "entropy-like" complexity measures, such as "approximate entropy". Other arbitrary specialized convenience functions can easily be defined in a couple lines of code.
+
+```@docs
+approx_entropy
+```
```

Comment:
Well now I am confused. I thought we agreed that we do this table thingy and therefore do not provide convenience functions anymore.

Reply:
> Well now I am confused. I thought we agreed that we do this table thingy and therefore do not provide convenience functions anymore.

We agreed on having exactly two convenience functions for complexity measures too, for educational purposes, like we do for entropy. Those would be sample entropy and approximate entropy, because they are the most widely used ones. Citing [yourself](https://github.com/JuliaDynamics/Entropies.jl/issues/133#issuecomment-1282262636) in #134: 

> Okay let's keep the sample_entropy and approxiamte_entropy as the only "convenience complexity functions".

Remaining complexity measures go in the table.

Reply:
I made this comment before the existence of the table was on the table however (pun not intended). If we have a table, which satisfies the "search engine" argument, do we need these convenience methods? Or, we can have a docs section that discusses the library design principles. I have one in Agents.jl as well. Okay. Leave the "convenience method" here. Once we have the table, I'll do a PR that proposes my section.